R/load.R

```diff
@@ -273,8 +273,19 @@ load_all <- function(path = ".",
     run_pkg_hook(package, "attach")
     run_user_hook(package, "attach")
 
+    if (export_all) {
```

Comment:
It has to happen later on than before because we now populate the package env, which is not yet created at this point.

I decided to populate it after the hooks are run for consistency with normal package loading. This way we'll catch programming errors earlier.

Reply:
Ok makes sense. Maybe it's worth extracting this out into a `setup_ns()` function? That'd make it a little more clear what arguments specifically affect the namespace.

Reply:
Good idea, I've now extracted `setup_pkg_env()` and `populate_pkg_env()`.